ci_framework/roles/reproducer/tasks/epel.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Enable CRB repositories (CentOS)
+  when:
+    - ansible_facts['distribution'] == 'CentOS'
+  become: true
+  ansible.builtin.command:
+    cmd: dnf config-manager --set-enabled crb
+
+- name: Enable CRB repositories (RHEL)
+  when:
+    - ansible_facts['distribution'] == 'RedHat'
+  become: true
+  vars:
+    arch: "{{ ansible_facts['architecture'] }}"
+  ansible.builtin.command:
```

Comment:
```suggestion
  community.general.rhsm_repository:
    name: "codeready-builder-for-rhel-9-{{ arch }}-rpms"
```

Reply:
I have a WIP on that. As said over last review call: don't bother on that file.